### DISCUSSION TITLE
Does ALE has syntax highlighting for its ALEHover?

### DISCUSSION THREAD

**Discussion by grech-ca** [w0rp/ale#4055] [Q&A - Ask for help with problems]
I wish I could see syntax highlighting when I reveal a type definition for my Typescript interface. Is it possible to do so in ALE?
![Screenshot 2022-02-06 at 13 54 58](https://user-images.githubusercontent.com/54347665/152677544-9c4960af-aaee-4754-ab30-cff1e8a388f2.png)

**Comment by w0rp** [comment]
It could probably be modified to use the same syntax highlighting as the file you're editing. I'm not that familiar with the floating windows, but it's probably just a few lines of code.
